var/spack/repos/builtin/packages/groff/package.py

```diff
@@ -42,6 +42,7 @@ class Groff(AutotoolsPackage, GNUMirrorPackage):
     # makeinfo is being searched for
     depends_on('texinfo', type='build', when='@1.22.4:')
     # configure complains when there is no uchardet that enhances preconv
```

Comment:
This comment applies to the uchardet dependency, can you keep the comment with the line?